providers/internetbs/api.go

```diff
@@ -0,0 +1,90 @@
+package internetbs
+
+import (
+	"encoding/json"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+// Api layer for Internet.bs
+
+type api struct {
+	creds struct {
```

Comment:
Why have this be a structure within the structure?  Could the fields be key and password instead of creds.key and creds.password?

Reply:
good point. fixed.